comparisontool/static/comparisontool/js/comparison-tool.js

```diff
@@ -2282,23 +2282,28 @@ var CFPBComparisonTool = (function() {
                     json_schools[key] = data;                        
                 });
                 json_schools = JSON.stringify( json_schools );
-                var request = $.ajax({
-                    type: "POST",
-                    url: posturl,
-                    dataType: "JSON",
-                    data: json_schools
-                });
-                request.done( function ( result ) {
 
-                });
-                request.fail( function ( xmlHttpRequest, textStatus ) {
-                    var foo = "";
-                    $.each(xmlHttpRequest, function(i, v) {
-                        foo += " " + i + ":" + v;
-                    });
-                    // alert( "Save failed!");
-                    $("#save-container").append( "Save failed!" + foo + " " + textStatus);
-                });
+                if ( global.worksheet_id !== "none") {
```

Comment:
Can we reverse the condition order (move the contents of the else inside the first block and the if to the second block)? This'll break https://eslint.org/docs/rules/no-negated-condition if we ever move this to a linted environment. If nothing after the if/else is applicable this could even be shortened to a guard statement:

```js
if ( global.worksheet_id === 'none' ) { 
  $("#save-container").append( "Save failed! (invalid worksheet id)" );
  return;
}
```